### PR TITLE
Sidekick from Readme

### PR DESCRIPTION
**Pull request by Fabi911**
i've added some text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Hallo,
+
+hier ist mein 3rster commit in diesem pranch.
+
+4 x 4 = 17
```
Comment (Miningmark) [inline README.md:5]: 4 x 4 = 16

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+6 x 3= 18
+
+5 x 0 = 5
```
Comment (Miningmark) [inline README.md:9]: 5 x 0 = 0

**Review by Miningmark — CHANGES_REQUESTED** [review]
some results were wrong

**Comment by Fabi911** [comment]
i've correct the results. 
When everything is ok, please approve.
@Miningmark

**Pull request merged**